doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -456,3 +456,9 @@
     let node = NodeBuilder::new().create::<ipc::Service>()?;
     let node = NodeBuilder::new().create::<local::Service>()?;
     ```
+
+## Thanks To All Contributors Of This Version
+
+* [Christian »elfenpiff« Eltzschig](https://github.com/elfenpiff)
+* [Mathias »elBoberido« Kraus](https://github.com/elboberido)
+* [»orecham«](https://github.com/orecham)
```

Comment:
@elBoberido Clicked merge a nanosecond before the notification for this came 😅

Reply:
No problem. Just thought it would be easier for upcoming releases to use the automation.

I also realized that the heading on the main README should probably be
> Thanks to the core contributors

Once we have more irregular contributions, it would make more sense